src/lib/swagger-express-ts/api-model-property.decorator.ts

```diff
@@ -13,12 +13,21 @@ export interface IApiModelPropertyArgs {
 export function ApiModelProperty(
   args?: IApiModelPropertyArgs
 ): PropertyDecorator {
-  return function(target: any, propertyKey: string | symbol) {
-    const propertyType = Reflect.getMetadata(
+  return function (target: any, propertyKey: string | symbol) {
+    let propertyType: any;
+    if (Reflect.getMetadata(
```

Comment:
- please extract the variable
- avoid to compare variable with undefined : harder to read and not really needed
- do not set to undefined an already undefined variable  // propertyType = undefined;

```
const metadata = Reflect.getMetadata("design:type", target, propertyKey);
if(metadata){
propertyType = metadata .toLowerCase().name();
}
```